packages/math-input/src/store/actions.ts

```diff
@@ -0,0 +1,187 @@
+import type {Key} from "../data/keys";
+import type {
+    KeypadConfiguration,
+    KeyHandler,
+    Cursor,
+    ActiveNodesObj,
+    Border,
+} from "../types";
+
+// naming convention: verb + noun
+// the noun should be one of the other properties in the object that's
+// being dispatched
+
+export const DismissKeypadActionType = "DismissKeypad";
```

Comment:
I felt like since it was a single file, it made sense to locate the actions closer to the rest of the redux stuff.

(I'm sorry about this, but...) it wasn't just a move, I also typed everything.

Reply:
Yeah, much nicer to place in this directory. I wouldn't expect to find the actions where they were.